### PR TITLE
Update readme for 0.7.1 release

### PR DESCRIPTION
**Pull request by piyush-garg**
# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if functionality changed/added)
- [ ] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [ ] Regenerate the manpages and docs with `make docs` and `make man` if needed.
- [ ] Run the code checkers with `make check`
- [ ] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/cli/blob/master/CONTRIBUTING.md)
for more details._

### CODE REVIEW

**Comment by chmouel** [comment]
/lgtm
/approve

Thanks 👍🏼🔥

File: README.md [hunk 1]
```diff
@@ -83,12 +83,12 @@ Download the latest binary executable for your operating system:
   dnf install tektoncd-cli
   ```
 
-  * [Binary RPM package](https://github.com/tektoncd/cli/releases/download/v0.6.0/cli_0.6.0_Linux-64bit.rpm)
+  * [Binary RPM package](https://github.com/tektoncd/cli/releases/download/v0.7.1/cli_0.7.1_Linux-64bit.rpm)
```
Comment (danielhelfand) [inline README.md]: https://github.com/tektoncd/cli/releases/download/v0.7.1/tektoncd-cli-0.7.1_Linux-64bit.rpm

File: README.md [hunk 2]
```diff
@@ -83,12 +83,12 @@ Download the latest binary executable for your operating system:
...
 
   On any other RPM based distros, you can install the rpm directly:
 
    ```shell
-    rpm -Uvh https://github.com/tektoncd/cli/releases/download/v0.6.0/cli_0.6.0_Linux-64bit.rpm
+    rpm -Uvh https://github.com/tektoncd/cli/releases/download/v0.7.1/cli_0.7.1_Linux-64bit.rpm
```
Comment (danielhelfand) [inline README.md]: https://github.com/tektoncd/cli/releases/download/v0.7.1/tektoncd-cli-0.7.1_Linux-64bit.rpm

File: README.md [hunk 3]
```diff
@@ -106,13 +106,13 @@ Download the latest binary executable for your operating system:
 
   The PPA may work with older releases, but that hasn't been tested.
 
-  * [Binary DEB package](https://github.com/tektoncd/cli/releases/download/v0.6.0/cli_0.6.0_Linux-64bit.deb)
+  * [Binary DEB package](https://github.com/tektoncd/cli/releases/download/v0.7.1/cli_0.7.1_Linux-64bit.deb)
```
Comment (danielhelfand) [inline README.md]: https://github.com/tektoncd/cli/releases/download/v0.7.1/tektoncd-cli-0.7.1_Linux-64bit.deb

File: README.md [hunk 4]
```diff
@@ -106,13 +106,13 @@ Download the latest binary executable for your operating system:
...
 
   On any other Debian or Ubuntu based distro, you can simply install the binary package directly with `dpkg`:
 
   ```shell
-  curl -LO https://github.com/tektoncd/cli/releases/download/v0.6.0/cli_0.6.0_Linux-64bit.deb
-  dpkg -i cli_0.6.0_Linux-64bit.deb
+  curl -LO https://github.com/tektoncd/cli/releases/download/v0.7.1/cli_0.7.1_Linux-64bit.deb
```
Comment (danielhelfand) [inline README.md]: https://github.com/tektoncd/cli/releases/download/v0.7.1/tektoncd-cli-0.7.1_Linux-64bit.deb

**Comment by chmouel** [comment]
/lgtm
/approve

Thanks 👍🏼🔥

**Pull request merged**